run-check.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
-runhaskell dumptypes.hs Prelude | typediff check/Prelude.types -
-runhaskell dumptypes.hs Data.Foldable | typediff check/Data.Foldable.types -
-runhaskell dumptypes.hs Data.List | typediff check/Data.List.types -
+runhaskell dumptypes.hs Prelude.Compat | typediff check/Prelude.Compat.types -
+runhaskell dumptypes.hs Data.Foldable.Compat | typediff check/Data.Foldable.Compat.types -
+runhaskell dumptypes.hs Data.List.Compat | typediff check/Data.List.Compat.types -
```

Comment:
Similarly, should a `Data.Traversable.Compat.types` file be created and checked here?